code/thrive-design-stylesheet-new.css

```diff
@@ -5312,7 +5312,7 @@ body.ribbit .btn.back-to-top:is(:hover, :focus) {
 
 #InteriorPageTitle::after {
     content: '';
-    height: 157px;
+    height: 70%;
```

Comment:
We don't want this btw, we want to keep the pixel height for Page Title as mentioned here: https://3.basecamp.com/3619374/buckets/32507769/card_tables/cards/7469638526#__recording_7471728854